### ISSUE TITLE
Random Items Tagged as Quest Items

### ISSUE THREAD

**Issue by Kinomora** [Zoinkwiz/quest-helper#1139]
![image](https://user-images.githubusercontent.com/9062578/236353249-1df267ed-620d-4085-9725-6ee805979c43.png)

I have quest cape and no active diary task to complete, yet when I have quest helper enabled random items are marked as "Quest Items" on the ground such as Law Runes in this example, but for other items too including Waterskins (4).

What is the cause of this?

**Comment by gwen-lily** [comment]
![image](https://github.com/Zoinkwiz/quest-helper/assets/14190748/7daadc6e-fd24-4b32-81bf-23fbdb306f58)

I have similar behavior, most recently this saw.

**Comment by gwen-lily** [comment]
Absolutely, this account (to this day) does not possess all the requirements for a few elite diaries, including Ardougne (where the behavior occurred)

**Comment by pajlada** [comment]
Do you have the "Always highlight needed items" setting enabled?

![image](https://github.com/Zoinkwiz/quest-helper/assets/962989/806b9b74-70f6-4cc6-9931-961f4251f312)
